src/edfi-performance-test/.mypy.ini

```diff
@@ -0,0 +1,14 @@
+[mypy]
+files=edfi_performance_test, tests
+
+[mypy-edfi_performance_test.*]
+ignore_missing_imports = True
```

Comment:
All local modules are resulting in mypy error - "installed but missing library stubs or py.typed marker local modules" unless I setup to ignore imports. Thoughts?